platform/analytics/google-analytics/build-process-measuring-pageviews-and-events.md

```diff
@@ -44,25 +44,68 @@ To support our React-based single page apps, we have turned on reporting of Hist
+    *   https://github.com/department-of-veterans-affairs/component-library/blob/master/src/helpers/test-helpers.js
+
+
+#### Forms System
+
+There are several analytics events baked into actions in the forms-system.  If you look in the primary actions.js file, you will see dataLayer pushes (via helper function recordEvent) for submission, successful submission, and others.  Keep in mind you will need to use a product prefix we have allowed in GTM.
+
+
+#### Static Sites pages
+
+Static content (such as Drupal content) on va.gov is run through an application called static-pages.  Because most of these pages can not use components from the component-library, we have implemented several application-wide event listeners for things like CTA buttons, Action Links, and other.  These can be found in this directory.
+
+One important development is the transition of the component-library from React components to web components.  These new web components should be able to be used in static pages.
+
+
+### Example:
+
+Your VFS team is building a new feature for the Profile page and wants to track how many times users submitted an information change for this section. You would structure this event like: 
+
+```js
+window.dataLayer.push({
+  event: 'profile-transaction',
+  'profile-section': 'edu-direct-deposit-information',
+  'profile-addressSuggestionUsed': 'no',
+});
+```
+Within Google Tag Manager, we would define a more readable format for an Event Category of "Transactions", Event Action of "EDU Direct Deposit Information", and an Event Label of "profile-transaction". 
+
+
+### Instrumentation:
 
 The code snippets takes on a format like `window.dataLayer.push({ event: 'vets-custom-event'})`. Here is the process we take to implement this tracking. 
 
 1. Analytics-Insights will guide you in the naming convention. 
-1. FE teams will implement this into code.  There is a [helper function that can be used](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/monitoring/record-event.js).
-1. In parallel, we update Google Tag Manager to pick up these events. We define an Event Category, Event Action, and Event Label that translates the dataLayer event into a more readable format for Google Analytics users. 
-1. Once the code is on staging, we will QA that it is working correctly. 
+2. FE teams will implement this into code.  There is a [helper function that can be used](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/monitoring/record-event.js).
+3. In parallel, we update Google Tag Manager to pick up these events. We define an Event Category, Event Action, and Event Label that translates the dataLayer event into a more readable format for Google Analytics users. 
+4. Once the code is on staging, we will QA that it is working correctly. 
 
 The naming convention for products begins with these prefixes:
```

Comment:
@bsmartin-ep Reference and link to [this list](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/google-analytics/build-process-measuring-pageviews-and-events.md#event-tracking)

Reply:
Not sure I understand @jonwehausen.  You want me to anchor to somewhere on the same page?